packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -74,9 +74,29 @@ export interface RoutesManifest {
 
 // The output bundle options are specified here
 export interface OutputBundleOptions {
+  /**
+   * bundleYamlPath: Path to the generated bundle.yaml file
```

Comment:
You can remove the `<variable>:` prefix here. 

Take a look at go/tsjs-style#jsdoc-general-form for how to format single line jsdocs

Reply:
done